tests/test_toga_backend_implementation/widgets/test_button_implementation.py

```diff
@@ -0,0 +1,105 @@
+import unittest
+
+
+# BUTTON DEFINITIONS
+# if the button core API changes it should be reflected here with a new
+# 'check_requirements_of_version_<new_version>' function.
+def check_requirements_of_version_0_1_0(self):
+    # Here we check if the button-implementation-requirements of version 0.1.0 are fulfilled.
+    # TODO implement a more robust way of checking if the requirements are meet. ast or re module maybe?
+    self.assertIn('class Button(', self.FILE)
+    self.assertIn('self._create()', self.FILE)
+    self.assertIn('def create(self):', self.FILE)
+    self.assertIn('def _set_label(self,', self.FILE)
+
+
+# TEST PLATFORM IMPLEMENTATIONS OF TOGA BUTTON
+class TestAndroidButtonImplementation(unittest.TestCase):
+    __version__ = '0.1.0'
```

Comment:
The ```__version__ ``` allows us to have different implementation versions. Some backends are maybe not as developed as others.